internal/pkgecosystem/ecosystem.go

```diff
@@ -19,3 +21,34 @@ var (
 func (p PkgManager) String() string {
 	return p.Name
 }
+
+// escape ensures that args being used are safely escaped for usage.
+func escape(s string) string {
```

Comment:
Rather than having to deal with escaping, could we just not invoke the analysis script/sandbox container via `bash -c`, and just pass arguments individually?

Reply:
This is a good idea. I will switch this over. I'd much rather pass the arguments as an array FWIW.

Reply:
Done.